### ISSUE TITLE
@default gemset not listed in manage rubies

### ISSUE THREAD

**Issue by mpapis** [remear/jewelrybox#114]
I see only @global on the list of gemsets, I see the gemset via gemsets.

**Comment by jgoodall** [comment]
This still seems to be an issue - is there a workaround?

**Comment by remear** [comment]
@jgoodall What exactly is it you're looking for and where are you looking for it? While that may seem a trivial question, you could be looking for @default in the Manage Rubies view, the Gemsets View, or the System Bar Menu.

**Comment by jgoodall** [comment]
In the manage rubies view.

Sent from a mobile device.

On Aug 10, 2012, at 9:47 AM, Ben Mills notifications@github.com wrote:

@jgoodall https://github.com/jgoodall What exactly is it you're looking
for and where are you looking for it? While that may seem a trivial
question, you could be looking for @default https://github.com/default in
the Manage Rubies view, the Gemsets View, or the System Bar Menu.

—
Reply to this email directly or view it on
GitHubhttps://github.com/remear/jewelrybox/issues/114#issuecomment-7642358.

**Comment by remear** [comment]
The Set As Default Ruby area will be removed from the Manage Rubies area. Click on the ruby name listed in the bottom left of the JewelryBox window or change your default ruby from the system bar menu.

**Comment by jgoodall** [comment]
how do you set 'default' to be the default gemset instead of 'global'?

**Comment by mpapis** [comment]
you can not change `default` gemset, default means without `@name`, @global
is the one inherited by all other gemsets including the one without name
(default).

**Comment by remear** [comment]
To avoid confusion, @global and @default have been explicitly listed in all pertinent areas. This will be available in v1.4.

**Comment by mpapis** [comment]
not sure if this is related, I was fixing `rvm gemset list`, and it shows now default:

``` log
gemsets for ruby-1.9.3-p194 (found in /home/mpapis/.rvm/gems/ruby-1.9.3-p194)
=> (default)
   rvm-test
```
